src/main/java/at/hannibal2/skyhanni/config/storage/ProfileSpecificStorage.java

```diff
@@ -303,6 +306,9 @@ public static class BitsStorage {
         public SimpleTimeMark boosterCookieExpiryTime = null;
     }
 
+    @Expose
+    public SimpleTimeMark godPotExpiryTime = SimpleTimeMarkFarPast();
```

Comment:
Could you elaborate more please

Reply:
Have not looked at this in IntelliJ. But this doesn't look like a valid java line. What is the default value of this variable?

Reply:
![image](https://github.com/user-attachments/assets/2bd7a9c0-c9cd-40eb-bbc6-91b5a5a5d9f7)

Reply:
Method naming does not follow Java convention. Methods typically start with a lowercase letter, e.g. simpleTimeMarkFarPast().

The method name is not very descriptive. Consider something like getFarPastSimpleTimeMark() to clarify its intent.

If GenericWrapper and getIt() are needed, consider renaming getIt() to something more meaningful.

Reply:
Also why is this wrapper needed?

Reply:
This is not a new thing, this has existed since June.
![image](https://github.com/user-attachments/assets/d026703e-3bdc-4eb6-b4dd-1440d5a17596)

I'm fine with changing it if you really want me to but this is not a new thing. And the wrapper is needed because you can't assign a Kotlin class value to a Java object unless it's got a static definition, which `SimpleTimeMark.farPast()` is not. It's the same reason that right below that, we have a wrapper for Duration - they're composed types that simplify to `long`, making them easy to use in the storage, but initial values have to be wrapped.

Reply:
can we use `new SimpleTimeMark(0)` instead then? really not a fan of this wrapping, and the function name breaks java conversations. so sad i have not noticed this before

Reply:
![image](https://github.com/user-attachments/assets/a204b378-210c-4494-940b-b52c33bcf301)

No we cannot.